lib/Jsrt/JsrtSourceHolder.h

```diff
@@ -17,6 +17,9 @@ namespace Js
         TUnloadCallback scriptUnloadCallback;
         JsSourceContext sourceContext;
 
+#ifndef NTBUILD
```

Comment:
This will cause build error on Full. See line 51.

---
In reply to: [85743215](https://github.com/Microsoft/ChakraCore/pull/1830#discussion_r85743215) [](ancestors = 85743215)